### PR TITLE
Add new consignment configuration parameters

### PR DESCRIPTION
**Pull request by kellynm**
Added generation_method and file_type as configuration parameters to improve clarity of the consignment generation options. Rearranged the config file hierarchy to separate parameter-based generation parameters from input fule parameters.

### CODE REVIEW

**Comment by kellynm** [comment]
@wenzeslaus I think this added configuration parameter to explicitly set the consignment generation method will improve clarity for user. Please let me know what you think.
Also, the linting test is failing related to yaml loader. I couldn't figure out how to fix. Could you check it and let me know what changes are needed?

**Comment by wenzeslaus** [comment]
> the linting test is failing related to yaml loader. I couldn't figure out how to fix. Could you check it and let me know what changes are needed?

It seems a new version of Pylint added that warning. The code is unrelated to your changes, but you can fix that by adding the following to the lines from the error message:

```python
# pylint: disable=no-value-for-parameter
```

So the whole thing will look like,

for line 389 in `popsborder/simulation.py`:

```python
    return yaml.load(text)  # pylint: disable=no-value-for-parameter
```

and for line 409:

```python
        return yaml.load(open(filename))  # pylint: disable=no-value-for-parameter
```

We can disable them, because it is a fallback code for older versions of PyYAML (and the new code is addressing the issue the warning is trying to raise).

File: docs/consignments.md [hunk 1]
```diff
@@ -1,33 +1,78 @@
...
- the `consignment` key in the configuration file.
+## Consignment generation method
+The consignments can be either purely synthetic or based on inspection records
+ (e.g., Form 280 or AQIM). Configuration for the consignments is under the
+ `consignment` key in the configuration file.
+
+ The `generation_method` key can be either `parameter_based` to generate
+ synthetic consignments based on user-provided list of parameter values, or
+ `input_file` to create consignments that match inspection records in a CSV
+ file.
```
Comment (wenzeslaus) [inline docs/consignments.md:11]: The leading space coming from the original text is just a typo as far as I can tell.

**Review by wenzeslaus — CHANGES_REQUESTED** [review]
This is great. I agree it makes the interface more clear and it fits how we organize other values in the configuration. The documentation improved as well.

There is a lot of unrelated changes in newlines in the notebooks and CSV files, but they are all in the right direction (`\r\n` to just `\n` and adding end of line at the end of file, i.e., generally more universal Unix style), so I'm happy to include it. Just mention that in the final commit message during merge.

My only major comment is about the in documentation. The other two are just stylistic.

File: docs/consignments.md [hunk 2]
```diff
@@ -1,33 +1,78 @@
...
+needs to be specified. 
+
+```yaml
+consignment:
+  items_per_box:
+    default: 200
+```
+
+Currently, no further settings for `items_per_box` are used, but in the future
+further settings might be added. For example, F280 and AQIM inspections records
```
[the remark this answers is not in the document]
Comment (kellynm) [reply docs/consignments.md:30]: I think this is just a mistake. It was written in the documentation already and I just rearranged it. I'll remove that statement.

**Pull request merged**